aries-site/src/pages/templates/side-to-side-navigation.mdx

```diff
@@ -1,3 +1,101 @@
-import { ComingSoon } from '../../components';
+import { Box, Image, NameValueList, NameValuePair } from 'grommet';
+import { DiagramPreview, Example } from '../../layouts';
 
-<ComingSoon />
\ No newline at end of file
+## When to use
+
+- Tabs to maneuver between relevant information within the same view.
+- Hamburger menus, sidebars, nav bars, wizards.
+- Switching from one service to another service.
+- Sidebar to navigate between pages.
+- Accordions to open a single panel to navigating between content within the same page.
+
+## HPE Local Header Example
+
+The HPE product’s local header which contains grouped, frequently used and top-level information.
+
+Example coming soon.
+
+## HPE Global Header Example
+
+The Global Header, also known as the HPE Common Header and Footer Web Service, provides a tunable, yet consistent header to be used across all HPE applications.
+
+For more information, reference the [Global Header page](/templates/global-header).
+
+### Guidelines
+
+- Global Header should be on all applications and websites
+- Allows users to move between top-level important destinations
+- Located at the top of all application and websites
+
+<Example 
+  previewWidth="xlarge"
+  height="xsmall"
+  plain
+>
+  <DiagramPreview
```

Comment:
I realize that the screenshot I have on Figma is probably not the best one as I didn't crop it and you can see the background color underneath. It also looks a little pixelated. I'll make a note of this for myself and replace the one in Figma. Once I've replaced it, i'll ping you.

Reply:
I actually pulled this from what we already have on the GlobalHeader page -- think the original screenshot came from a Windows screen which may be reason for seeming a little pixelated.

If you get an updated version, let me know and I'll replace.

Reply:
Got it! What i'll do is recreate it in Figma as a vector that way it performs better in different resolutions.